src/Perspex.Base/Data/InstancedBinding.cs

```diff
@@ -30,7 +30,8 @@ public class InstancedBinding
         /// The value used for the <see cref="BindingMode.OneTime"/> binding.
         /// </param>
         /// <param name="priority">The binding priority.</param>
-        public InstancedBinding(object value, BindingPriority priority = BindingPriority.LocalValue)
+        public InstancedBinding(object value,
+            BindingPriority priority = BindingPriority.LocalValue)
```

Comment:
Sorry. The changes on this were from the previous design I had for propagating validation status back. I'll fix that.